wcmc_components/config/routes.rb

```diff
@@ -1,4 +1,10 @@
 WcmcComponents::Engine.routes.draw do
   post '/', to: 'table#list'
   post '/download', to: 'table#download'
+
```

Comment:
A confusing thing I noticed, but don't understand why. When curling the following routes, countries.json returns a no routes matched error, but the others don't... (`curl -X 'POST' URL_GOES_HERE`)

countries.json - no routes
countries - all good
countries/api/v1.json - all good
countries/api/v1 - all good

Any ideas why that might be?

I think this may have been a pre-existing thing, so sorry to bring it up now!